Tests/L0/SonarQubePostTest/PRCA/PostCommentsTests.ps1

```diff
@@ -589,22 +596,25 @@ Unregister-Mock GetModifiedFilesInPR
 # Test 3 - Post a code flow style message
 #
 $mockDiscussionClient = InitPostCommentsModule $false
-Register-Mock GetModifiedFilesInPR { @("some/path1/file.cs") }
+Register-Mock GetModifiedFilesInPR { @("some/path1/file.cs", "path/not/in/changes/response") }
 Register-Mock GetCodeFlowLatestIterationId 
-Register-Mock GetCodeFlowChanges
-Register-Mock TryGetCodeFlowChangeTrackingId {$true}
```

Comment:
The tests didn't catch this because I mocked too much stuff. TryGetCodeFlowChangeTrackingId  doesn't make client OM API calls, so it needn't be mocked.